### PR TITLE
Check the version of the package.

### PR DESCRIPTION
**Pull request by aplanas**
Also adds a new dependency to `python-semantic_version`

### CODE REVIEW

**Comment by aplanas** [comment]
Do not merge yet, I will use a rpm's bind instead `python-semantic_version`

**Comment by aplanas** [comment]
* Remove sematic_version dependency and use rpm binding `labelCompare`

File: update_crawler.py [hunk 1]
```diff
@@ -59,13 +90,21 @@ def get_source_packages(self, project, expand=False):
...
 
     def get_source_verifymd5(self, project, package):
         """ Return the verifymd5 of a source package."""
         root = ET.fromstring(self._get_source_package(project, package))
         return root.get('verifymd5')
 
+    def get_source_version(self, project, package):
+        """ Return the version of a source package."""
+        root = ET.fromstring(self._get_source_package(project, package))
+        return stringToVersion(root.find('version').text)
```
Comment (lnussel) [inline update_crawler.py]: are you sure this is needed? Looks like obs returns version and release in separate tags

**Comment by aplanas** [comment]
* Use the `version` and `release` returned from OBS

**Pull request merged**